### PR TITLE
20170729 v1 tab scroll

### PR DESCRIPTION
**Pull request by hsukfry88**
20170729 v1 tab scroll

### CODE REVIEW

File: index.ios.js [hunk 1]
```diff
@@ -5,17 +5,62 @@ import React, {
...
+  },
+});
+
+const onChangeTabFn = (obj) => {
+  console.log(obj);
+};
 
 export default class BeeUI extends Component {
   render() {
+    const tabArray = [{ tabLabel: '全部' }, { tabLabel: '为审阅' }, { tabLabel: '待审阅' }];
```
Comment (charlzyx) [inline index.ios.js]: tabArray todel

File: index.ios.js [hunk 2]
```diff
@@ -5,17 +5,62 @@ import React, {
...
+            </View>
+            <View tabLabel="已审阅">
+              <Text>已审阅</Text>
+            </View>
+          </TabScene>
+          <TabScene
+            tabDefaultText={{ color: '#fff' }}
+            tabActiveText={{ color: '#888' }}
+            tabUnderLineStyle={{ backgroundColor: '#000' }}
+            tabBarBarbackground={'#82A0FA'}
```
Comment (charlzyx) [inline index.ios.js]: background / Height => style

File: src/TabScene/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+### TabScene组件: 特别提醒如果TabScene不在最顶层，请在顶层设置<View style={{flex:1}}>否则下面的content容器不显示
+
+| 参数名称 | 描述 | 类型 | 属性值 | 默认值|
+|---|---|---|---|---|
+| tabBarPosition | 设置tabBar的布局位置，顶部或底部 | `String` | `top` |
+| initialPage | 设置初始时activeTab的key | `Number` | `0` |
```
Comment (charlzyx) [inline src/TabScene/README.md]: page => sence

File: src/TabScene/README.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+| onChangeTab | callback函数 | `function` | (obj)=>{} | `{ i:currentKey, from: prevKey, ref: dom }`
+| renderTabBar | 要渲染的tabBar组件,通过向该函数传递自定义组件来修改默认的tabBarCompontent |`func` | 默认自动渲染`TabBarCompontent` |
+| scrollWithoutAnimation | 动画开关 | `bool` | `true` |
+| tabActiveText | 焦点文字颜色样式 | `Text.propTypes.style` |`{color: #82A0FA }`|
```
Comment (charlzyx) [inline src/TabScene/README.md]: check style if only color, think style

File: src/Button/index.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import React, {
```
Comment (hsukfry88) [inline src/Button/index.js]: 改为Touch

**Pull request merged**